runtime/server_http_response_test.go

```diff
@@ -163,7 +163,7 @@ func TestCallWriteJSONWithBadJSON(t *testing.T) {
 		),
 	)
 
-	resp, err := gateway.MakeRequest("GET", "/foo", nil)
+	resp, err := gateway.MakeRequest("GET", "/foo", nil, nil)
```

Comment:
This method signature is starting to take a lot of arguments. If we add any more we should consider accepting an Options argument that contains theses fields. That was when you call Make request
```
MakeRequest(&Options{
   method: "Get",
   path: "/foo"
}) 
```
which gives us better readability and nullable arguments.